tests/integration/construct_component_configure_provider/go/go.mod

```
@@ -1,6 +1,6 @@
 module github.com/pulumi/pulumi/tests/integration/construct_component_configure_provider
 
-go 1.23.0
+go 1.24.2
```

```suggestion
go 1.23.0
```
Don't think this needs to change with the README even if it is due an update.